assets/js/main.js

```diff
@@ -266,95 +265,43 @@ function modal() {
     });
 }
 
+function elasticSearch(query, callback) {
+    var base_url = 'https://advisorysg.ent.ap-southeast-1.aws.found.io';
+    var engine = 'ghost';
+    var search_only_key = 'search-brya3eig6n5g9ybimkw3o9u3';
+
+    var oReq = new XMLHttpRequest();
```

Comment:
What does `oReq` stand for?

Reply:
Changed it to a more meaningful name, searchReq